server/static/js/admin/controllers.js

```diff
@@ -533,6 +533,29 @@ app.controller("SubmissionListCtrl", ['$scope', '$stateParams', '$window', 'Sear
     $scope.search = function() {
       $scope.getPage($scope.currentPage)
     }
+    
+    $scope.download_zip = function(query, all, courseId) {
+      filename = 'query_(your email)_(current time).zip'
+      Search.download_zip({
+        query: query,
+        all: all,
+        courseId: courseId
+      }, function() {
+        $window.swal({
+          title: 'Success',
+          text:'Saving submissions to ' + filename +
+          '\n Zip of submissions will be ready in Google Cloud Storage ok_grades_bucket in a few minutes',
+          type: 'success',
+          confirmButtonText: 'View zip',
+          cancelButtonText: 'Not now',
+          showCancelButton: true},
+          function() {
+            $window.location = 'https://console.developers.google.com/storage/browser/ok_grades_bucket/';
```

Comment:
Could we just give them the direct url to the file and avoid redirecting them? Not everyone will have privileges to view the file. Either way not a big deal because we'll come up with a better way to proxy files through ok-server. 

I believe the format is : 
`https://console.developers.google.com/m/cloudstorage/b/ok_grades_bucket/o/{FILENAME}`